### ISSUE TITLE
Pixels not rendering past a certain point on larger displays

### ISSUE THREAD

**Issue by cubedmeatgoeshere** [stephengolub/ESPHome-display-generator#7]
See attached pic

![image](https://user-images.githubusercontent.com/96257138/150146132-b0128b1c-5e4d-4072-a7b0-10cd10c10b88.png)

I really like this project but it currently cannot be used for common e-ink displays since they typically have many pixels (296x128 in my case)

**Comment by stephengolub** [comment]
I suspect #6 and this issue are related. I put this together as a quick POC and haven't had an opportunity to optimize it.

I'll take a look shortly.

**Comment by cubedmeatgoeshere** [comment]
I see it's likely because of the maxWidth and maxHeight variables in code.js. I've changed them to 296 and 128 respectively and it runs fine for me! #6 may prevent any larger allowances to be unstable however.

![image](https://user-images.githubusercontent.com/96257138/150151429-bb993fe7-014f-4f92-ab77-8cb681e87fbe.png)

**Comment by stephengolub** [comment]
let me try something

**Comment by stephengolub** [comment]
Yep. You were right. I opened #8 to address it. Can you pull that down and see if that helps?
